### PR TITLE
Set Kubevirt SRIOV lane mandatory

### PR DESCRIPTION
**Pull request by ormergi**
Kubevrit SRIOV lane should be mandatory and block a given PR if it fails.

Now that SRIOV lane is stabilized and all flaky tests are resolved it is possible to set 
it mandatory.

Please share if there are any problems we should fix first before merging this PR.

### CODE REVIEW

**Comment by qinqon** [comment]
/hold
sriov is still failing at ping https://prow.apps.ovirt.org/?job=pull-kubevirt-e2e-kind-1.17-sriov&state=failure

But the other tests are passing allright, let's skip those test and mark the lane as mandatory.

**Comment by oshoval** [comment]
Once what Quique said will be fix, i don't know about other problems with the current situation.
But i do have problems with the still not merged PR 
https://github.com/kubevirt/kubevirtci/pull/492
Wrote there my comments, as this lane will be come mandatory and there might be issue with 492
please consider them.

**Comment by EdDev** [comment]
> But i do have problems with the still not merged PR (but approved)
> [kubevirt/kubevirtci#492](https://github.com/kubevirt/kubevirtci/pull/492)
> Wrote there my comments, as this lane will become mandatory and there might be issues with [kubevirt/kubevirtci#492](https://github.com/kubevirt/kubevirtci/pull/492)
> please consider them.

I do not see the relevance to this change.
Your issues with a non merge PR belongs there, not here.

**Comment by oshoval** [comment]
> > But i do have problems with the still not merged PR (but approved)
> > [kubevirt/kubevirtci#492](https://github.com/kubevirt/kubevirtci/pull/492)
> > Wrote there my comments, as this lane will become mandatory and there might be issues with [kubevirt/kubevirtci#492](https://github.com/kubevirt/kubevirtci/pull/492)
> > please consider them.
> 
> I do not see the relevance to this change.
> Your issues with a non merge PR belongs there, not here.

I understand, but this is why i wrote its already approved and that once this lane will become mandatory,
it will be more important.
I just asked to raise the importance of it please.

**Comment by oshoval** [comment]
https://github.com/kubevirt/kubevirtci/issues/484
This i saw one time, don't know if / how many times more it happened

**Comment by qinqon** [comment]
@ormergi @EdDev I think we can mark as mandatory the lane.

**Review by EdDev — APPROVED** [review]

**Pull request merged**
